src/Sidekick.Presentation.Blazor.Electron/Views/ViewLocator.cs

```diff
@@ -105,7 +108,7 @@ public async Task Open(View view, params object[] args)
             var pathArgs = new StringBuilder();
             foreach (var arg in args)
             {
-                pathArgs.Append($"/{JsonSerializer.Serialize(arg).EncodeBase64()}");
+                pathArgs.Append($"/{HttpUtility.UrlEncode(JsonSerializer.Serialize(arg).EncodeBase64())}");
```

Comment:
Like we talked about on discord, this needs to be centralized somewhere (Probably Sidekick.Extensions)

Reply:
Cleaned up, moved to String Extensions.